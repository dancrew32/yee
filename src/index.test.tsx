import React from "react";
import renderer from "react-test-renderer";

jest.mock("./main.less", () => jest.fn());
jest.mock("./home.md", () => jest.fn());

import { App } from "./index.tsx";

describe("<App />", () => {
  it("should snapshot", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
