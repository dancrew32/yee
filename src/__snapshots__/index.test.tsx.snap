// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should snapshot 1`] = `
Array [
  <nav>
    <a
      className="active"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className=""
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
    <a
      className=""
      href="/products"
      onClick={[Function]}
    >
      Products
    </a>
  </nav>,
  <div>
    Loading...
  </div>,
]
`;
